AppleMusicClone/Views/MainView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MainView.swift
+//  AppleMusicClone
+//
+//  Created by Hankyu Lee on 2022/07/05.
+//
+
+import SwiftUI
+
+struct MainView: View {
+    var body: some View {
+        TabView {
+            
+            //TODO: 각 뷰 넣기
+            Text("1")
+                .tabItem {
+                    Image(systemName: "play.circle.fill")
+                    Text("지금 듣기")
+                }
+            
+            Text("2")
+                .tabItem {
+                    Image(systemName: "dot.radiowaves.left.and.right")
+                    Text("라디오")
+                }
+
+            Text("3")
+                .tabItem {
+                    Image(systemName: "play.circle.fill")
+                    Text("지금 듣기")
```

Comment:
```suggestion
                    Image(systemName: "folder")
                    Text("보관함")
```
보관함 symbol이 없어서 이렇게 사용하셨을까요?!
![folder@2x](https://user-images.githubusercontent.com/59143479/177702751-accd67b6-a7d1-42e2-9bc3-0e3d500665c9.png)
`folder` symbol 정도도 괜찮을 것 같습니다

Reply:
앗 감사합니다 수정하겠습니다!